plugins/catalog-backend/src/processing/types.ts

```diff
@@ -28,6 +29,7 @@ import {
 export type EntityProcessingRequest = {
   entity: Entity;
   state?: JsonObject; // Versions for multiple deployments etc
+  credentials?: BackstageCredentials;
```

Comment:
Think that we can drop this when we move all the permissions to the router instead.